### PR TITLE
Deploy PGPool to kubernetes.

### PR DESCRIPTION
**Pull request by dadiorchen**

### CODE REVIEW

File: pgpool/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# PGPool settings
+
+### To generate the config by kustomize and apply to kubernates
+
+```
+kubectl --kubeconfig=../dev-k8s-treetracker-kubeconfig.yaml kustomize deployment/overlays/development/ | kubectl apply --kubeconfig=../dev-k8s-treetracker-kubeconfig.yaml -f -
```
Comment (mckornfield) [inline pgpool/README.md:6]: is this path right? `dev-k8s-treetracker-kubeconfig.yaml`

File: pgpool/deployment/base/pgpool_configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pgpool-config
+  labels:
+    app: pgpool-config
+#    backend_hostname0 = 'db-postgresql-sfo2-nextgen-do-user-1067699-0.db.ondigitalocean.com'
```
Comment (mckornfield) [inline pgpool/deployment/base/pgpool_configmap.yaml:7]: should this be deleted?

File: pgpool/deployment/base/treetracker-pgpool-deployment.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      labels:
+        app: "treetracker-pgpool"
+    spec:
+      containers:
+      - name: "treetracker-pgpool"
+        image: pgpool/pgpool:4.2.2
+        env:
+        - name: POSTGRES_USERNAME
+          value: "xxxx"
+#          valueFrom:
```
Comment (mckornfield) [inline pgpool/deployment/base/treetracker-pgpool-deployment.yml:24]: delete the secrets or? Should they be secret values/ sealed secrets?

File: pgpool/deployment/overlays/test/pgpool_configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pgpool-config
+  labels:
+    app: pgpool-config
+#    backend_hostname0 = 'db-postgresql-sfo2-nextgen-do-user-1067699-0.db.ondigitalocean.com'
```
Comment (mckornfield) [inline pgpool/deployment/overlays/test/pgpool_configmap.yaml:7]: same comment as before

**Comment by dadiorchen** [comment]
The newest code resolves #41

**Comment by ZavenArra** [comment]
@dadiorchen in this case, please click 'resolve conversation' on MattK's comments

**Comment by dadiorchen** [comment]
Didn't notice that function, done, thanks

**Pull request merged**